dowhy/causal_refuters/add_unobserved_common_cause.py

```diff
@@ -44,6 +44,14 @@ def __init__(self, *args, **kwargs):
         :param effect_fraction_on_treatment: float: If effect_strength_on_treatment is not provided, this parameter decides the effect strength of the simulated confounder as a fraction of the effect strength of observed confounders on treatment. Defaults to 1.
         :param effect_fraction_on_outcome: float: If effect_strength_on_outcome is not provided, this parameter decides the effect strength of the simulated confounder as a fraction of the effect strength of observed confounders on outcome. Defaults to 1.
         :param plotmethod: string: Type of plot to be shown. If None, no plot is generated. This parameter is used only only when more than one treatment confounder effect values or outcome confounder effect values are provided. Default is "colormesh". Supported values are "contour", "colormesh" when more than one value is provided for both confounder effect value parameters; "line" when provided for only one of them.
+        :param simulated_methods_name: method type to add unobserved common cause. "linear_based" by default."PartialR2" for linear sensitivity analysis
+        :param percent_change_rvalue: percentage reduction for robustness value
```

Comment:
Can you say more for the docstring. Is it the percentage reduction that user wants? What happens when it is low or high--how to set this value?